src/helpers/connection.ts

```diff
@@ -0,0 +1,45 @@
+import { type ChainName, getBaseUrl } from "./chains.js";
```

Comment:
Actually, this file should be renamed to config. But this contains the limited bit of information the SDK needs to know to configure connections and behavior. There are only 3 settings, and each one is only relevant in certain contexts, so they are also their own interfaces. This allows us to be pretty lazy about checks. For instance, if you are only ever going to _query_ tableland, then you don't need any signer information, and vice versa.

Reply:
Narrator, he did rename it.